### PR TITLE
[9.0] port - stock_location_area_data

### PR DESCRIPTION
**Pull request by dvdhinesh**
ported "stock_location_area_data" to 9.0

### CODE REVIEW

File: stock_location_area_data/README.rst [hunk 1]
```diff
@@ -1,12 +1,10 @@
-Features
-========
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (pedrobaeza) [inline stock_location_area_data/README.rst:1]: Use latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: stock_location_area_data/__openerp__.py [hunk 2]
```diff
@@ -1,29 +1,13 @@
...
+# (C) 2015 Savoir-faire Linux, Dhinesh D
+
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 {
     'name': 'Stock Location Area Data',
-    'version': '8.0.0.1.0',
-    'author': 'Savoir-faire Linux',
-    'maintainer': 'Savoir-faire Linux',
+    'version': '9.0.0.1.0',
```
Comment (pedrobaeza) [inline stock_location_area_data/__openerp__.py]: Version 9.0.1.0.0

File: stock_location_area_data/__openerp__.py [hunk 3]
```diff
@@ -1,29 +1,13 @@
...
+    'author': 'Savoir-faire Linux, Dhinesh D',
+    'maintainer': 'Savoir-faire Linux, Odoo Community Association (OCA)',
```
Comment (pedrobaeza) [inline stock_location_area_data/__openerp__.py]: OCA should be in author list

File: stock_location_area_data/data/stock_location_area_data.xml [hunk 4]
```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<!-- 
+    (C) 2015 Savoir-faire Linux, Dhinesh D
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
   <data>
```
Comment (pedrobaeza) [inline stock_location_area_data/data/stock_location_area_data.xml]: You can remove <data> tag

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks for the submission @dvdhinesh - Only issue I see is the outdated ReadMe template that @pedrobaeza noted.

**Comment by dvdhinesh** [comment]
@lasley Readme files are updated to the latest convention. Thanks for pointing it out.

**Comment by dvdhinesh** [comment]
@pedrobaeza Could you restart this Travis build (https://travis-ci.org/OCA/stock-logistics-warehouse/builds/164564588). This was failed because the problem we faced couple of days before I guess.

@lasley Could you apply the needs review label?

**Review by lasley — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**